lib/console.rb

```diff
@@ -0,0 +1,9 @@
+class Console
+  def retrieve_user_input
+    gets
+  end
+
+  def print_message(string)
+    print(string)
```

Comment:
In ruby, `print` does not add a new line after a string, and `puts` does. Did you consider this when implementing this?